src/client/components/stockDataRegion/stockDataRegion.css

```diff
@@ -1,4 +1,5 @@
 .stockDataRegion {
   display: flex;
-  flex-direction: column;
+  flex: 1;
+  padding: 20px 52px;
```

Comment:
* spends his whole software career learning that random undescriptive non-constant numbers like 20 and 52 are bad

* uses random numbers for everything because css

Reply:
@gregdardis check it https://sass-lang.com/guide#topic-2

bet ya wanna learn SASS now